parm/config/config.resources

```diff
@@ -422,6 +473,12 @@ elif [ $step = "eobs" -o $step = "eomg" ]; then
     export nth_eobs=2
     if [[ "$machine" = "WCOSS_DELL_P3" ]]; then export nth_eobs=7; fi
     export npe_node_eobs=$(echo "$npe_node_max / $nth_eobs" | bc)
+    #The number of tasks and number of CPUs used must be the same for eobs
+    #For S4, this is accomplished by running 10 tasks/node
+    if [[ "$machine" = "S4" ]]; then
+       export npe_node_eobs=10
+       export wtime_eobs="00:30:00"
+    fi
```

Comment:
based on the calculation on Line 475, for the different partitions of S4, the result on 479 will either be 16 or 10.  Whats wrong with those?
Is the 15min for eobs too short on S4?